pkg/endpoint/bpf.go

```diff
@@ -1146,3 +1147,62 @@ func (e *Endpoint) RequireRouting() (required bool) {
 func (e *Endpoint) RequireEndpointRoute() bool {
 	return e.DatapathConfiguration.InstallEndpointRoute
 }
+
+type netNSManager interface {
```

Comment:
Should this just be part of datapath.Datapath? (alternatively, some sort of DatapathOps? I would expect to capture more things into it someday, like the `bpf.LocalMapPath` wrappers at the start of this function).

Reply:
Oh, the `d` here is not the daemon... and piping a `Datapath` to here won't work. sigh... I don't suppose I could convince you to make this a callback instead of of adding all the boilerplate?

Reply:
What do you envision for the callback exactly? I'm unsure where I would plumb it into `pkg/workloads`. Can you provide an example of what you want?

Reply:
Can't we just call `connector.ConfigureNetNSForIPVLAN()` without the indirection via the interface/struct from `(*Endpoint) FinishIPVLANInit()`?

Reply:
I just meant instead of passing around `netNSManager` objects pass about `func (netNsPath string) (mapFD, mapID int, err error)` directly. This is definitely a style thing. I just don't like types that just exist to pass functions about.

Reply:
Got it, makes sense!

Reply:
Done!